File: CODES/4) Php/3) BOOLEAN/22.php
<?php
$n = 432;
echo " N = " . $n . "<br>";
$yuz = intval($n / 100);
$on = intval($n / 10) % 10;
$bir = $n % 10;
$l = ($yuz < $on and $on < $bir or $yuz > $on and $on > $bir);
if ($l) {
    echo "True";
} else {
    echo "False";
}

File: LANGUAGES/1) C++/3) BOOLEAN/22.cpp
#include <iostream>

using namespace std;

int main()
{
    bool l;
    int yuz, on, bir, n;
    cout << " n = ";
    cin >> n;
    yuz = n / 100;
    on = n / 10 % 10;
    bir = n % 10;
    l = (yuz < on and on < bir or bir < on and on < yuz);
    cout << l;
    return 0;
}